File: MyApplicationconRuben/app/src/main/java/com/example/myapplicationconborja/MainActivity.java
package com.example.myapplicationconborja;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private Button button = null;
    private TextView textView = null;
    private EditText editText = null;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        button = (Button) findViewById(R.id.button);
        textView = (TextView) findViewById(R.id.textView);
        editText = (EditText) findViewById(R.id.editTextNumber);


    }
    public void onClick (View v){
        String textViewText = myTextView.getText().toString();
        String esPrimo = getString(R.String.text_)
    }
}

File: ElMejorJuego/app/src/main/java/com/example/elmejorjuego/MainActivity.kt
package com.example.elmejorjuego

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.CheckBox
import android.widget.RadioButton
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val monster: RadioButton = findViewById(R.id.monster)
        val lol: RadioButton = findViewById(R.id.league)
        val titanfall : RadioButton = findViewById(R.id.titanfall)
        val civ6: RadioButton = findViewById(R.id.civilization)
        val texto:TextView = findViewById(R.id.textoGuay)
        val check:CheckBox = findViewById(R.id.darle)

        monster.setOnClickListener{
            texto.text = getString(R.string.monster )
        }

        lol.setOnClickListener{
            texto.text=getString(R.string.lol)
        }

        titanfall.setOnClickListener{
            texto.text=getString(R.string.titan_fall)
        }

        civ6.setOnClickListener{
            texto.text=getString(R.string.civ)
        }

        check.setOnClickListener{
            if (check.isChecked){
                texto.text=getString(R.string.checked)
            } else{
                texto.text=getString(R.string.unchecked)
            }

        }

    }
}